index.js

```diff
@@ -27,16 +27,26 @@ Bin.prototype.objectAt = function (collection, index) {
   return collection[index];
 };
 
-// abstract
+// abstract: return coordinates of element at index.
```

Comment:
in the future, we should just port there to the YUIDoc so some convention covers the doc creation. Unless this is the convention.